packages/mui-base/src/Composite/Root/useCompositeRoot.ts

```diff
@@ -170,7 +172,9 @@ export function useCompositeRoot(params: UseCompositeRootParameters) {
           }
 
           if (nextIndex !== activeIndex && !isIndexOutOfBounds(elementsRef, nextIndex)) {
-            event.stopPropagation();
```

Comment:
@atomiks Why is `stopPropagation()` needed here, and is it ever ok to force propagate it? This breaks a [test](https://github.com/mui/base-ui/pull/751)

Given this structure:
```jsx
<Tabs.Root onKeyDown={handleKeyDown}>
  <Tabs.List>
    <Tabs.Tab value={0} />
    <Tabs.Tab value={1} />
  </Tabs.List>
  {/* tab panels */}
</Tabs.Root>
```
`Tabs.List` is a `CompositeRoot`, so on arrow keys, the event doesn't propagate and `handleKeyDown` doesn't get called

Reply:
I'm not certain on that. It may not be necessary in most cases